e2e-test-server/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/GoogleCloudPlatform/opentelemetry-operations-go/e2e-test-server
+
+go 1.16
+
+require (
+	cloud.google.com/go/pubsub v1.12.0
+	github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace v0.20.1
```

Comment:
This is building from the tag right? It should be building from the current source checkout for CI runs.

Reply:
Fixed